packages/manager/apps/dedicated/client/app/billing/billing-feature-availability.js

```diff
@@ -5,7 +5,7 @@ export default class {
   }
 
   allowOrderTracking() {
-    return this.deny('US');
+    return this.allow('CA', 'EU', 'US');
```

Comment:
```suggestion
    return true;
```
even better we could maybe remove `if` where this is applied

Reply:
What if we decided to introduce a new region? 

Looks good to me.